### PR TITLE
Added the ability to eat organs, mmm tastes like people

### PR DESCRIPTION
**Pull request by Henry12116**
## About the PR
I added the ability to eat basically all organs in the game. 

## Why / Balance
[Issue 20163](https://github.com/space-wizards/space-station-14/issues/20163)
Some organs already have a food attribute (e.x. `BaseHumanOrgan`), which gives them the option in game to be eaten, but nothing happens when it is pressed.

As for balance, one concern is that this makes it very easy to hide bodies that have been gibbed.

## Technical details
Updated all `.yml` files under `Resources/Prototypes/Body/Organs/<thing>.yml` so that the things respective organs are now edible. Every organ is comprised of 5u nutriment. It would be cool if stomachs were comprised of what ever the individual had been eating, but this can be iterated on.

Additionally, I had to update some values in the cargo livestock catalog to prevent arbitrage. The additional nutrients per organ increased the animals value.

## Media
https://streamable.com/4y4dqc

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
No breaking changes.

**Changelog**

:cl:
- add: You can now eat most organs. Bon appetit!

### CODE REVIEW

File: Resources/Prototypes/Catalog/Cargo/cargo_livestock.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     sprite: Mobs/Animals/bee.rsi
     state: 0
   product: CrateNPCBee
-  cost: 4000
+  cost: 4125
   category: Livestock
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/Cargo/cargo_livestock.yml:8]: Just round these up.

**Comment by deltanedas** [comment]
> As for balance, one concern is that this makes it very easy to hide bodies that have been gibbed.

you can already juice organs so it isnt much easier as youll have to wait for your own stomach to empty before eating the victims one

**Comment by Henry12116** [comment]
@metalgearsloth @deltanedas could one of you rerun the checks? I am unable to.

**Comment by deltanedas** [comment]
`git commit --allow-empty -m test; git push`

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**